lib/stat_tracker.rb

```diff
@@ -303,4 +375,18 @@ def fewest_goals_scored(team_id)
   def average_win_percentage(team_id)
     win_percentage_per_team[team_id]
   end
+
+  def worst_season(team_id)
+    lowest_win_percent = win_percentage_by_season(team_id).values.min
+    win_percentage_by_season(team_id).invert[lowest_win_percent]
+  end
```

Comment:
Given that helper methods aren't working as expected I'm not sure how this method works! will review it again once those are updated.